modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * understanding_openshift_gitops/about-redhat-openshift-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-openshift-go-common-terms_{context}"]
+= Glossary of common terms for OpenShift {gitops-shortname}
+
+This glossary defines common OpenShift {gitops-shortname} terms.
+
+Argo CD::
+An open-source declarative tool that automates the continuous deployment (CD) of Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+{gitops-shortname}::
+{gitops-shortname} is a declarative way to implement continuous deployment for cloud native applications. In {gitops-shortname}, a Git repository contains deployment resources, which Argo CD then keeps synchronizing with its cluster state.
```

Comment:
Suggestion: Discard if you feel otherwise:

In {gitops-shortname}, a Git repository contains deployment resources, which Argo CD then keeps synchronizing with its cluster state. -> In {gitops-shortname}, deployment resources are stored in a Git repository, which Argo CD keeps synchronizing with its cluster state.

Reply:
@Dhruv-Soni11 Retaining the content as is, with active voice. Thanks for the suggestion, though.